build.sc

```diff
@@ -250,6 +250,10 @@ object contrib extends MillModule {
     }
    }
 
+  object tut extends MillModule {
+    def moduleDeps = Seq(scalalib)
+    def testArgs = Seq("-DMILL_VERSION=" + build.publishVersion()._1)
```

Comment:
should this be `_.2` like it is in other parts of the build file?

Reply:
I tried that, but unfortunately in a project with local changes it just meant that Coursier was unable to resolve the dependency (as MILL_VERSION is some unpublished version of mill)